src/clients/UBAClient/UBAClientWithRefresh.ts

```diff
@@ -3,30 +3,24 @@ import winston from "winston";
 import { HubPoolClient, SpokePoolClient } from "..";
 import { BaseUBAClient } from "./UBAClientBase";
 import { updateUBAClient } from "./UBAClientUtilities";
-import { RelayFeeCalculator, RelayFeeCalculatorConfigWithMap } from "../../relayFeeCalculator";
+import { RelayFeeCalculatorConfigWithMap } from "../../relayFeeCalculator";
 import { UBAChainState } from "./UBAClientTypes";
 export class UBAClientWithRefresh extends BaseUBAClient {
-  /**
-   * The RelayFeeCalculator is used to compute the relayer fee for a given amount of tokens.
-   */
-  protected readonly relayCalculator: RelayFeeCalculator;
-
   // @dev chainIdIndices supports indexing members of root bundle proposals submitted to the HubPool.
   //      It must include the complete set of chain IDs ever supported by the HubPool.
   // @dev SpokePoolClients may be a subset of the SpokePools that have been deployed.
   constructor(
     readonly chainIdIndices: number[],
     readonly tokens: string[],
     protected readonly hubPoolClient: HubPoolClient,
-    protected readonly spokePoolClients: { [chainId: number]: SpokePoolClient },
+    readonly spokePoolClients: { [chainId: number]: SpokePoolClient },
```

Comment:
I think we should be explicit with the typing. 
```suggestion
    public readonly spokePoolClients: { [chainId: number]: SpokePoolClient },
```